python-sdk/nuscenes/eval/predict/config.py

```diff
@@ -10,9 +10,11 @@ class PredictionConfig:
 
     def __init__(self,
                  metrics: List[Metric],
-                 seconds: int = 6):
+                 seconds: int = 6,
+                 frequency: int = 2):
         self.metrics = metrics
         self.seconds = seconds
+        self.frequency = 2
```

Comment:
Should be `self.frequency = frequency`, no?

Reply:
Should also add units (Hz) for frequency

Reply:
Yes